src/pages/PassportRedemption/TrackScreen.tsx

```diff
@@ -97,19 +98,6 @@ const Track = ({ setCurrentScreenView }: Props) => {
     },
   }))(Tooltip);
 
-  const socialMediaLinks = [
```

Comment:
just curious if there's a reason why we don't include the email in these social media links but we include them in the links for the footer

Reply:
It wasn't on the designs cc @nanxiy